apps/meteor/ee/app/models/server/models/LivechatPriority.js

```diff
@@ -7,9 +7,8 @@ export class LivechatPriority extends Base {
 	constructor() {
 		super('livechat_priority');
 
-		this.tryEnsureIndex({ name: 1 }, { unique: true });
+		this.tryEnsureIndex({ name: 1 });
```

Comment:
Please tell alek how to remove the unique index on their DB. Don't think we need a migration since we're not in develop yet

Reply:
alright, removing this change